src/themes/default/components/core/blocks/MyAccount/MyOrder.vue

```diff
@@ -137,13 +143,36 @@
 </template>
 
 <script>
+import Vue from 'vue'
 import MyOrder from '@vue-storefront/core/compatibility/components/blocks/MyAccount/MyOrder'
 import ReturnIcon from 'theme/components/core/blocks/Header/ReturnIcon'
+import ProductImage from 'theme/components/core/ProductImage'
+import { getThumbnailPath } from '@vue-storefront/core/helpers'
+import { mapActions } from 'vuex'
 
 export default {
   mixins: [MyOrder],
   components: {
-    ReturnIcon
+    ReturnIcon,
+    ProductImage
+  },
+  data () {
+    return {
+      itemThumbnail: []
+    }
+  },
+  methods: {
+    ...mapActions({
+      getProduct: 'product/single'
+    })
+  },
+  beforeMount () {
```

Comment:
i'd do that on mounted, so there is no way to have problems with SSR
```suggestion
  mounted () {
```